ISPN-12622 Expose Cross-Site Views via CLI and REST endpoint (#9008)
infinispan/infinispan: cli/src/main/resources/help/site.adoc

`site clear-push-state-status --cache=myCache` +
Clears the status of the push state operation for `mycache`.

`site name` +

> Returns the local site name. If cross-site replication isn't available, it returns the word "local".

Reviewer: ```suggestion
Returns the name of the local site. If cross-site replication is not configured, the name of the local site is always "local".
```

`site view` +

> Returns a list with all the site's name, or an empty list ("[]") if cross-site replication isn't available.

Reviewer: ```suggestion
Returns a list of names for all sites or an empty list ("[]") if cross-site replication is not configured.
```
